app/controllers/application_controller.rb

```diff
@@ -91,7 +91,8 @@ def load_cms_site
   def load_cms_content
     return if admin_path?
 
-    @cms_page ||= Comfy::Cms::Page.find_by_full_path(request.original_fullpath.gsub(%r{\A/#{I18n.locale}/?}, '/'))
+    @cms_page ||= Comfy::Cms::Page.find_by_full_path(request.original_fullpath
```

Comment:
I would refactor this slightly so it's more readable.
More specifically, creating a local variable with the value that goes into `find_by_full_path` and naming it properly would also help understand what that is exactly.